jd4/case.py

```
@@ -256,20 +256,72 @@ def read_legacy_cases(config, open):
                           memory_bytes,
                           int(score_str))
 
-def read_yaml_cases(config, open):
-    for case in yaml.safe_load(config)['cases']:
-        if 'judge' not in case:
-            yield DefaultCase(partial(open, case['input']),
-                              partial(open, case['output']),
-                              parse_time_ns(case['time']),
-                              parse_memory_bytes(case['memory']),
-                              int(case['score']))
+def read_yaml_cases(config, open, zip_file):
+    try:
+        cfg = yaml.safe_load(config)
+    except:
```

Why ignore exceptions here?